.github/workflows/update-package-request.yaml

```diff
@@ -0,0 +1,102 @@
+name: update-package-request
+
+on:
+  # When a calling repository opens an issue with metadata,
+  # this will call spack to retrieve it and open a pull request with changes.
+  # This will automate external packages from needing to manually update packages
+  issues:
+    types: [opened, edited]
+
+jobs:
+  update-package:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    - name: Parse Update Issue
+      env:
+        title: ${{ github.event.issue.title }}
+        number: ${{ github.event.issue.number }}
+      run: |  
+        printf "Issue title: ${title}\n"
+        printf "Issue number: ${number}\n"
+        echo "${{ github.event.issue.body }}" > new-issue.txt
+        cat new-issue.txt
+        # Script to parse the issue, sets envars needed for subsequent workflows
+        wget https://raw.githubusercontent.com/sciworks/spack-updater/main/parse_issue.py
```

Comment:
Is there any reason we can't bring this script into the repo directly? I don't like having our CI pull a moving-target from github, that will be unnecessarily brittle.

Reply:
I think the way to do this is to make the script an action and to pin it at a commit, as we do for all other external actions.